### PR TITLE
Make testgrid updater pass lint

### PR DESCRIPTION
**Pull request by fejta**
/assign @michelle192837 @krzyzacy @BenTheElder

### CODE REVIEW

File: testgrid/cmd/updater/main.go [hunk 1]
```diff
@@ -121,17 +123,24 @@ type Started struct {
...
 	Metadata   Metadata `json:"metadata"`
 	running    bool
 }
 
-// infra-commit, repos, repo, repo-commit, others
+// Metadata holds the finished.json values in the metadata key.
+//
+// Metadata values can either be string or string map of strings
+//
+// Special values: infra-commit, repos, repo, repo-commit, others
```
Comment (michelle192837) [inline testgrid/cmd/updater/main.go]: Is there documentation to link to for these values?

File: testgrid/cmd/updater/main.go [hunk 2]
```diff
@@ -121,17 +123,24 @@ type Started struct {
...
 type Metadata map[string]interface{}
 
+// String results the name key if its value is a string.
```
Comment (michelle192837) [inline testgrid/cmd/updater/main.go]: returns?

File: testgrid/cmd/updater/main.go [hunk 3]
```diff
@@ -288,17 +306,20 @@ func extractRows(buf []byte, meta map[string]string) (map[string][]Row, error) {
 	return rows, nil
 }
 
+// ColumnMetadata holds key => value mapping of metadata info.
 type ColumnMetadata map[string]string
 
+// Column represents a build run, which includesone or more row results and metadata.
```
Comment (michelle192837) [inline testgrid/cmd/updater/main.go]: nit: includes one

File: testgrid/cmd/updater/main.go [hunk 4]
```diff
@@ -307,6 +328,7 @@ type Row struct {
 	Icon     string
 }
 
+// Overall calculcates the generated-overall row value for the current column
```
Comment (michelle192837) [inline testgrid/cmd/updater/main.go]: nit: calculates

File: testgrid/cmd/updater/main.go (hunk 1 above)
Comment (fejta) [reply testgrid/cmd/updater/main.go]: Not externally, and not sure we want to keep any of this

File: testgrid/cmd/updater/main.go (hunk 2 above)
Comment (fejta) [reply testgrid/cmd/updater/main.go]: Done

File: testgrid/cmd/updater/main.go (hunk 3 above)
Comment (fejta) [reply testgrid/cmd/updater/main.go]: Done

File: testgrid/cmd/updater/main.go (hunk 4 above)
Comment (fejta) [reply testgrid/cmd/updater/main.go]: Done

File: testgrid/cmd/updater/main.go (hunk 1 above)
Comment (michelle192837) [reply testgrid/cmd/updater/main.go]: Fair enough.

**Comment by BenTheElder** [comment]
whoo!

**Pull request merged**